internal/operators/ocs/ocs_operator.go

```diff
@@ -125,33 +125,36 @@ func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 func (o *operator) GetHostRequirements(_ context.Context, cluster *common.Cluster, host *models.Host) (*models.ClusterHostRequirementsDetails, error) {
 	numOfHosts := len(cluster.Hosts)
 
-	var disks int64 = 0
+	var diskCount int64 = 0
 	if host.Inventory != "" {
```

Comment:
what happens here if host.Inventory is empty?

Reply:
if Inventory is empty, for compact mode it will return the requirements with [one disk,](https://github.com/openshift/assisted-service/blob/master/internal/operators/ocs/ocs_operator.go#L138) and for minimal mode, it will return the requirement with [no disk](https://github.com/openshift/assisted-service/blob/master/internal/operators/ocs/ocs_operator.go#L173)